policy/modules/services/virt.te

```diff
@@ -143,6 +143,9 @@ mls_trusted_object(virt_log_t)
 type virt_runtime_t alias virt_var_run_t;
 files_runtime_file(virt_runtime_t)
 
+type virt_runtime_common_t alias virt_common_var_run_t;
```

Comment:
I'd prefer to have it named `virt_common_runtime_t`. Also, no need for the alias since there is nothing to be backwards compatibility issue.